frontend/stmt_inferrer.py

```diff
@@ -296,12 +279,30 @@ def _init_func_context(args, context, solver):
     return local_context, args_types
 
 
+def annotated(node):
+    """Check the arguments and return are annotated in a function definition"""
+    if not node.returns:
+        return False
+    for arg in node.args.args:
```

Comment:
As we discussed in the meeting, methods using generic type variables are not supported.